src/lib/organisms/methodsHeader.svelte

```diff
@@ -28,7 +24,11 @@
             <h2 class="bottom">Voorbeelden</h2>
           </a>
         </li>
-      {/each}
+
+        {#if method.pdf}
+          <a href="{method.pdf.url}" download="{method.slug}.pdf" target="blank" class="button-pdf">Download</a>
```

Comment:
the classname _button-pdf_ doesn't look DRY. Is there a _button_ class you can use? Add an extra _pfd_ class for specific styling.

Reply:
@Daniquedejong had je hier al naar gekeken?